sd-validation/jsonld.js

```diff
@@ -0,0 +1,71 @@
+/**
+ * @license Copyright 2018 Google Inc. All Rights Reserved.
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License. You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions and limitations under the License.
+ */
+'use strict';
+
+const walkObject = require('./helpers/walkObject');
+
+const CONTEXT = '@context';
+const KEYWORDS = [
+  '@base',
+  '@container',
+  CONTEXT,
+  '@graph',
+  '@id',
+  '@index',
+  '@language',
+  '@list',
+  '@nest',
+  '@none',
+  '@prefix',
+  '@reverse',
+  '@set',
+  '@type',
+  '@value',
+  '@version',
+  '@vocab',
+];
+
+/**
+ * @param {string} fieldName
+ * @returns boolean
+ */
+function validKeyword(fieldName) {
+  return KEYWORDS.includes(fieldName);
+}
+
+/**
+ * @param {string} name
+ * @returns {string | null} error
+ */
+function validateField(name) {
```

Comment:
is 'field' a json ld term? i didnt see it used in https://json-ld.org/spec/latest/json-ld/

Reply:
Fixed.